src/server.ts

```diff
@@ -16,16 +18,25 @@ import Cookies from "universal-cookie";
 const server: ApolloServer = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req }): object => {
-    // to get token from cookies
+  context: async ({ req }): Promise<object> => {
     const cookies = new Cookies(req && req.headers.cookie);
-    const cookieToken = cookies.get("RCTC_USER");
-    // token for backwards compatibility
+    const cookieToken: string = cookies.get("RCTC_USER");
     const fallbackToken = req && req.headers.authorization;
-
+    const token: string = cookieToken ? cookieToken : fallbackToken;
```

Comment:
change `cookieToken ? cookieToken : fallbackToken` to `cookieToken || fallbackToken`